cmd/kaniko-docker/main.go

```diff
@@ -90,9 +90,9 @@ func main() {
 			Usage:  "enable auto generation of build tags",
 			EnvVar: "PLUGIN_AUTO_TAG",
 		},
-		cli.BoolFlag{
+		cli.StringFlag{
```

Comment:
Better to use PLUGIN_CONFIG which is used by drone-plugins/drone-docker as well to specify the docker config json file

Reply:
https://github.com/drone-plugins/drone-docker/blob/master/cmd/drone-docker/main.go#L228